src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/JobsConstants_fr.properties

```diff
@@ -32,4 +33,5 @@ jobListLabel=Les {0} travaux suivants sont toujours en cours d''exécution.
 terminateRunningJobsCaption=Terminer les travaux en cours d''exécution
 terminateJobsCaption=Terminer les travaux
 jobsTabLabel=Onglet des travaux
-viewAllJobsTitle=Voir tous les travaux
\ No newline at end of file
+viewAllJobsTitle=Voir tous les travaux
+launcherTitle=Launcher
```

Comment:
Is this the right French translation?

Reply:
No, but that's ok. These will all get reviewed and (re)-localized when we hand it off. I've made an effort to provide the French strings in some cases, but otherwise it's fine for us to just put in English as a placeholder (in general).